8Knot/pages/repo_overview/visualizations/ossf_scorecard.py

```diff
@@ -76,7 +85,7 @@ def toggle_popover(n, is_open):
 
 # callback for ossf scorecard
 @callback(
-    Output(f"{PAGE}-{VIZ_ID}", "children"),
+    [Output(f"{PAGE}-{VIZ_ID}", "children"), Output(f"{PAGE}-{VIZ_ID}-updated", "children")],
```

Comment:
I used to have it that way when I initially made this MR and it caused the MR to fail CI because of the linter.

Running the commit hooks manually caused the linter to make it one line. (See my commit named "format")

Reply:
Ah weird not a fan but not worth the fight atm

Reply:
want me to open a new issue to track this and potentially fix the CI rules?

I suspect its linting by line length, rather than always breaking up lists

Reply:
hmm lets open an issue and come back to this later. thank you for looking into this!

Reply:
Actually seeing what the linter to the line below if its an easy switch and fixes the part below lets just change it

Reply:
hmm. either my testing is wonky or its being inconsistent. I'll open an issue

Reply:
https://github.com/oss-aspen/8Knot/issues/732